lib/logstash/plugin.rb

```diff
@@ -111,9 +111,9 @@ def clear_watchdog
 
   public
   def inspect
-    if !@config.nil?
-      description = @config \
-        .select { |k,v| !v.nil? && (v.respond_to?(:empty?) && !v.empty?) } \
+    if !@params.nil?
+      description = @params \
```

Comment:
no, just keeping with the current format, some might like an explicit line break, I have no strong feelings about it